build/BuildValidator.proj

```diff
@@ -0,0 +1,63 @@
+<Project Sdk="Microsoft.Build.NoTargets">
```

Comment:
Can we name this `LocalizationValidator` or something like that/

I know the package is not named super well, but we can do better with the project.